constants.xml

```diff
@@ -50,6 +50,8 @@
         <MAX_NEIGHBORS_PER_ROUND>10</MAX_NEIGHBORS_PER_ROUND>
         <NUM_NODE_INCR_DIFFICULTY>5000</NUM_NODE_INCR_DIFFICULTY>
         <MAX_SHARD_NODE_NUM>7500</MAX_SHARD_NODE_NUM>
+        <HEARTBEAT_INTERVAL_IN_SECONDS>10</HEARTBEAT_INTERVAL_IN_SECONDS>
+        <HEARTBEAT_NUM_DS_EPOCHS_TIMEOUT>5</HEARTBEAT_NUM_DS_EPOCHS_TIMEOUT>
```

Comment:
For the case of RejoinAsDS, as we can't guarantee that after this num of ds epoch this node is still in DS committee, I think the current way of implementing RejoinAsDS may not work for the DS being kicked out within let's say the current 5 epochs

Reply:
Technically since we check the DS committee info before attempting RejoinAsDS, it can still work. But anyway I will consider this based on the other comment below about this constant.